conditional/templates/attendance_cm.html

```diff
@@ -1,4 +1,7 @@
 {% extends "nav.html" %}
+{% block title %}
+Committee Attendance
```

Comment:
Ummm... Something, something, didn't I just merge a pull request to change this to Directorship meetings.